src/main/java/frc/robot/commands/intake/IntakeFlip.java

```diff
@@ -46,6 +45,6 @@ public void end(boolean interrupted)
   @Override
   public boolean isFinished()
   {
-    return true;
+    return false;
```

Comment:
Why do we set this to `false`?  Doesn't that mean that the command will never exit?  We want this command to run once, then exit, every time it is triggered by the operator.

Reply:
There's one exception where you might want to use `false`, and that would be a default command such as `Drive` that needs to run forever.  If another command requires the drivetrain, it will just interrupt `Drive`, go do the other command, and then come back and keep running `Drive`.

Reply:
Basically if you want a command to be running forever in the background by default, you can use `true`, otherwise if it's more of a one-shot thing you can you `false`.

Reply:
Thanks, that was a dumb mistake on my part.